docs/sources/migration-guide/migrating-from-thanos-or-prometheus.md

```diff
@@ -47,53 +47,35 @@ aws s3 cp <DIRECTORY> s3://<TENANT>/<DIRECTORY>
 gsutil -m cp -r <DIRECTORY> gs://<TENANT>/<DIRECTORY>
 ```
 
-## Migrating the block `meta.json` metadata using `metaconvert`
+## Block metadata
 
-Every block has a `meta.json` metadata file used by Grafana Mimir, Prometheus, and Thanos to identify the block contents.
+Each block has a `meta.json` metadata file that is used by Grafana Mimir, Prometheus, and Thanos to identify the block contents.
 Each project has its own metadata conventions.
-The `metaconvert` tool migrates the `meta.json` metadata from one project to another.
 
-### Downloading `metaconvert`
+In the Grafana Mimir 2.1 (or earlier) release, the ingesters added an external label to the `meta.json` file to identify the tenant that owns the block.
+If you enable the sharding feature of Mimir's split-and-merge compactor, compactor adds a special label to identify the compactor-shard.
```

Comment:
This sentence looks like related to 2.1 or earlier only.

Reply:
Next paragraph says it still applies for 2.2:

> The compactor-shard label is still used, if sharding of blocks during compaction is enabled.

Reply:
I know but people sometimes just stop reading. Given the split-and-merge compactor shard label doesn't change by Mimir versions, I wouldn't mention it along with 2.1 or 2.2 changes at all.

Reply:
> Given the split-and-merge compactor shard label doesn't change by Mimir versions, I wouldn't mention it along with 2.1 or 2.2 changes at all.

OK.